src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -548,33 +548,37 @@ def _predict_with_probabilities(self, input_dataset, batch_size=None,
                 batch_data = batch.data[0]
                 size = batch_size
 
-            ctx0 = ctx[:batch_data.shape[0]] if batch_data.shape[0] < len(ctx) else ctx
-            split_data = _mx.gluon.utils.split_and_load(batch_data, ctx_list=ctx0, even_split=False)
+            num_devices = min(batch_data.shape[0], len(ctx))
+            split_data = _mx.gluon.utils.split_and_load(batch_data, ctx_list=ctx[:num_devices], even_split=False)
             classes = self.classes
 
+            from turicreate import SArrayBuilder as _SArrayBuilder
```

Comment:
There is no reason that I'm aware of to ever hide imports if they are being done at the function level.

Reply:
Fixed.